plugins.d/alarm-notify.sh

```diff
@@ -218,7 +219,7 @@ fi
 images_base_url="https://registry.my-netdata.io"
 
 # curl options to use
-curl_options=
+curl_options="--insecure"
```

Comment:
This PR should not change the default behavior of all cURL based notification mechanisms, especially if it's reintroducing a significant security issue (if cURL isn't checking certificates, it becomes trivial to MitM the connections and then ensure that notifications sent using cURL never arrive).

If MS Team integration _really_ requires cURL to not check certificates (and I seriously doubt that it can't be made to work with proper certificate checking), then you should be explicitly passing this option when you call cURL from `send_team()` _and_ making sure it's well documented that you are doing this.

Reply:
my bad, I was getting issue related to this #3489 on Centos Linux. I gonna remove this setup.